config/kibana.yml

```diff
@@ -109,6 +109,5 @@
 # metrics. Minimum is 100ms. Defaults to 5000.
 #ops.interval: 5000
 
-# The default locale. This locale can be used in certain circumstances to substitute any missing
-# translations.
-#i18n.defaultLocale: "en"
+# The locale. This locale will be used for translations.
```

Comment:
__nit:__ maybe something like this instead `Specifies locale to be used for all localizable strings, dates and number formats`?